dccs/blender/generate_protocol.bat

```diff
@@ -1,2 +1 @@
-python -m grpc_tools.protoc -Icrates/lgn-editor-proto/protos/ --python_out=dccs/blender/generated --grpc_python_out=dccs/blender/generated crates/lgn-editor-proto/protos/resource_browser.proto
 python -m grpc_tools.protoc -Icrates/lgn-editor-proto/protos/ --python_out=dccs/blender/generated --grpc_python_out=dccs/blender/generated crates/lgn-editor-proto/protos/source_control.proto
```

Comment:
We probably missed something in here, should we fix blender's client in this pr as well?

Reply:
If we don't, we're having a chat anyway with @kirdaybov to take ownership of the Blender part while he's on leave, so we might fix it later.